documentation/docs/pages/developers/nymvpncli.mdx

```diff
@@ -0,0 +1,278 @@
+  Older Debian/Ubuntu versions need to manually install `protobuf-compiler` >= v3.21.12
+</Callout>
+
+### Clone & `make`
+```sh
+git clone https://github.com/nymtech/nym-vpn-client.git
+cd nym-vpn-client/
+make
+```
+
+## Mnemonic Generation
+Create an account at [nymvpn.com](nymvpn.com) to obtain your mnemonic.
+
+## Start the daemon
+```sh
+sudo ./PATH/TO/nym-vpnd
+```
+
+If you are running for the first time you will see the following:
+
+```sh
+2024-12-11T11:03:58.202159Z  INFO nym_vpnd::environment: Setting up environment by discovering the network: mainnet
+2024-12-11T11:03:58.202205Z  INFO nym_vpn_network_config::discovery: No discovery file found, writing default discovery file
+2024-12-11T11:03:59.905505Z  INFO nym_vpnd::command_interface::start: Starting command interface
+2024-12-11T11:03:59.905660Z  INFO nym_vpnd::service::vpn_service: Starting VPN service
+2024-12-11T11:03:59.905879Z  INFO nym_vpnd::command_interface::start: Starting socket listener on: /var/run/nym-vpn.sock
+2024-12-11T11:03:59.906227Z  INFO nym_vpn_account_controller::controller: Starting account controller
+2024-12-11T11:03:59.906285Z  INFO nym_vpn_account_controller::controller: Account controller: data directory: "/var/lib/nym-vpnd/mainnet"
+2024-12-11T11:03:59.906313Z  INFO nym_vpn_account_controller::controller: Account controller: credential mode: false
+2024-12-11T11:03:59.913215Z  INFO nym_vpnd::command_interface::listener: Removed previous command interface socket: "/var/run/nym-vpn.sock"
+2024-12-11T11:03:59.977206Z  INFO nym_vpnd::service::vpn_service: VPN service initialized successfully
+2024-12-11T11:03:59.979246Z  INFO nym_vpn_account_controller::controller: Account id: (unset)
+2024-12-11T11:03:59.979265Z  INFO nym_vpn_account_controller::controller: Device id: BZWA5MRnEvRYD8WWrH9KULdj2Q1uTssu6idjgWFae9dv
+2024-12-11T11:03:59.979762Z  INFO nym_vpn_account_controller::storage: Ticketbooks stored: 0
+2024-12-11T11:03:59.982125Z  INFO nym_vpn_account_controller::controller: Received command: UpdateAccountState
+2024-12-11T11:03:59.982181Z  INFO nym_vpn_account_controller::shared_state: Setting mnemonic state to NotStored
+2024-12-11T11:03:59.982200Z  WARN nym_vpn_account_controller::commands: Returning error: NoAccountStored
+2024-12-11T11:03:59.982218Z  INFO nym_vpn_account_controller::controller: Received command: UpdateDeviceState
+2024-12-11T11:03:59.982230Z  INFO nym_vpn_account_controller::shared_state: Setting mnemonic state to NotStored
+2024-12-11T11:03:59.982240Z  WARN nym_vpn_account_controller::commands: Returning error: NoAccountStored
+```
+
+Ignore the `NoAccountStored` errors: these will disappear after the next step. **Leave the daemon running and run the following commands in another terminal window** or create an init file for `nym-vpnd`.
+
+## Run VPN
+We have to first store the account we have created online:
+```sh
+./PATH/TO/nym-vpn-cli store-account --mnemonic "<MNEMONIC_FROM_GENERATION_STEP>"
```

Comment:
I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`